### PR TITLE
Update History.rst with missing PR numbers

### PR DESCRIPTION
**Pull request by Nythiennzo**
I added all PR numbers in commits from the beginning.
I did not add those with failed checks.
I added everything at the top of other changes
Hope it helps.

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -48,6 +48,12 @@ Bug Fixes:
 
 Other Changes:
 
+* Update sponsoring section, add backers, thanks to `@hackebrot`_ (#950)
+* Changed link title for cookiecutter-django-paas, thanks to `@mpachas`_ (#933)
+* Merge pull request from stevepiercy/patch-6, thanks to `@hackebrot`_ (#911)
```
Comment (insspb) [inline HISTORY.rst]: 911
Another typos fix

File: HISTORY.rst [hunk 2]
```diff
@@ -48,6 +48,12 @@ Bug Fixes:
...
+* Fix Windows installation instructions, thanks to `@stevepiercy`_ (#887)
+* Fix the doc string in is_copy_only_path funciton for PEP257, thanks to `@harshtheworkman`_ (#742)
```
Comment (insspb) [inline HISTORY.rst]: 742 is issue, not PR. What PR was mentioned here?
is_copy_path is in PR 996, already in history.

File: HISTORY.rst [hunk 3]
```diff
@@ -48,6 +48,12 @@ Bug Fixes:
...
+* Add test_requirements.txt, thanks to `@ramnes`_ (#377)
```
Comment (insspb) [inline HISTORY.rst]: 377 is issue, not PR. Related PR 945 already in docs.

File: HISTORY.rst [hunk 4]
```diff
@@ -59,7 +65,7 @@ Other Changes:
 * Update doc string for ``is_copy_only_path()`` according to pep257, thanks to
   `@mathagician`_ and `@terryjbates`_ (#935, #949)
 * Update doc strings in ``cookiecutter/extensions.py`` according to pep257,
-  thanks to `@meahow`_ (#996)
+  thanks to `@meahow`_ (#996, #742)
```
Comment (insspb) [inline HISTORY.rst]: You mention one issue in two places. 
Such commits are hard to review. It is not correct at all.
One mention of PR itself is enough to find all history of the issue.

File: HISTORY.rst [hunk 5]
```diff
@@ -163,6 +169,7 @@ Bug Fixes:
 
 Other Changes:
 
+* Merge pull request from ruflin/remove-beat-generator, thanks to `@hackebrot`_ (#893)
```
Comment (insspb) [inline HISTORY.rst]: Template deletion. 
Trivial

File: HISTORY.rst [hunk 6]
```diff
@@ -247,6 +254,15 @@ Bug Fixes:
 
 Other Changes:
 
+* Merge pull request from stevepiercy/patch-3, thanks to `@hackebrot`_ (#870)
```
Comment (insspb) [inline HISTORY.rst]: trivial typos

File: HISTORY.rst [hunk 7]
```diff
@@ -247,6 +254,15 @@ Bug Fixes:
...
+* Mark test as xfail, thanks to `@hackebrot`_ (#850)
```
Comment (insspb) [inline HISTORY.rst]: 850 is issue. Reffered to PRs 856, 860. Both already in docs.

File: HISTORY.rst [hunk 8]
```diff
@@ -247,6 +254,15 @@ Bug Fixes:
...
+* Added the first case study, thanks to `@pydanny`_ (#708)
```
Comment (insspb) [inline HISTORY.rst]: 708 is issue. PR is 735. Already in docs.

**Comment by insspb** [comment]
Anyway, thank you for the dedicated time to it. 
Unfortunately currently requires full rework.

**Comment by kishan3** [comment]
@insspb Hi. What needs to be done I see there is `history.md` file now. Please guide.

**Pull request closed without merging**